examples/dhcpd.conf

```diff
@@ -0,0 +1,79 @@
+##
+##  ISC-DHCPD configuration
+##
+
+# General Config
+
+default-lease-time 86400;
+max-lease-time 86400;
+
+
+# Define Mitel Specific Options
+
+option space sipdect;
+
+option sipdect.omm-ip code 10 = ip-address;
+option sipdect.omm-ip2 code 19 = ip-address;
+option sipdect.omm-syslog-ip code 14 = ip-address;
+option sipdect.omm-syslog-port code 15 = unsigned integer 16;
+option magic_str code 224 = text;
+option tftp-list code 150 = array of ip-address;
```

Comment:
Defined but not used. (Yes, I sound like a compiler…)
What's the difference between this and `next-server`?

Reply:
Mitels Manual isn't very clear about this, at least for me. I will have to read it again and try to make this clear.